### DISCUSSION TITLE
Tree组件如何自定义层级缩进宽度或者说.ant-tree-indent的宽度如何修改还能保持层级结构

### DISCUSSION THREAD

**Discussion by aaronlin08** [ant-design/ant-design#42917] [Q&A]
贵工具好，我们目前使用了Tree组件，但是使用中发现层级之间的缩进太宽了，想缩小缩进的宽度，且保留层级结构。于是我们找到了.ant-tree-indent样式类，但是发现如果修改覆盖他，比如设置一个统一的宽度，那就无法体现层级结构了。请问我们这种需求，需要如何调整代码呢？
<img width="219" alt="屏幕截图 2023-06-08 105114" src="https://github.com/ant-design/ant-design/assets/39662519/3d5582c5-6df5-49e2-a2c6-528a509592dc">
<img width="199" alt="屏幕截图 2023-06-08 105154" src="https://github.com/ant-design/ant-design/assets/39662519/31d569ff-0d73-4233-9214-c4a128fc63dd">
最后，非常感谢~~
Best Regards
------------------------
myEmail:61092754@qq.com

**Comment by linxianxi** [comment]
我改了下可以啊，只是右边这种连接线的有问题
<img width="1417" alt="image" src="https://github.com/ant-design/ant-design/assets/47104575/ecd8fa41-f5eb-468e-9e73-f09652a42604">

**Reply by aaronlin08** [reply]
我去，回复的好快！谢谢你，雷锋！之前没有注意到内部还有ant-tree-indent-unit这个类样式，非常感谢！！！

**Comment by afc163** [comment]
我应该要给个变量。

**Reply by goldEli** [reply]
现在可以定义变量了吗？
